Gemfile

```diff
@@ -63,6 +63,7 @@ end
 group :test do
   # Use system testing [https://guides.rubyonrails.org/testing.html#system-testing]
   gem 'capybara'
+  gem 'rspec-rails'
   gem 'selenium-webdriver'
 end
 
```

Comment:
![image](https://github.com/Arch-Noize/rorblog/assets/93039359/9968267f-2155-4253-b083-ced8622b27b6)

- [ ] It would be nice if you give attention to the links highlighted in the image above 👀 because these links are broken 😨 Kindly remove them if you don't have any dedicated section in your readme file for these links or fix them to make your readme descriptive and professional 👍